sound/soc/sof/sof-spi-dev.c

```diff
@@ -22,19 +22,7 @@
 #include "hw-spi.h"
 #include "ops.h"
 
-/* FIXME: replace with some meaningful values */
-static struct snd_soc_acpi_mach spi_machines[] = {
-	{
-		.id = "INT343A",
-		.drv_name = "bxt_alc298s_i2s",
-		.sof_fw_filename = "intel/sof-spi.ri",
-		.sof_tplg_filename = "intel/sof-spi.tplg",
-		.asoc_plat_name = "0000:00:0e.0",
-	},
-};
-
```

Comment:
This might change yes, as far as I understand one of possibilities is to actually represent the Sue Creek as a codec with SPI controls and I2S audio, but since we aren't that far yet, I was using nocodec for testing.